src/lib.rs

```diff
@@ -39,15 +38,29 @@ pub trait CommandPtraceSpawn {
 
 impl CommandPtraceSpawn for Command {
     fn spawn_ptrace(&mut self) -> Result<Child> {
-        let child = self.before_exec(|| {
-            // Opt-in to ptrace.
-            ptrace(PTRACE_TRACEME, 0, ptr::null_mut(), ptr::null_mut())?;
-            Ok(())
-        }).spawn()?;
+        let child = unsafe {
```

Comment:
N.B.: This unsafe is required because of a deprecation: `before_exec` is now deprecated, and its replacement, `pre_exec`, is correctly marked as `unsafe`.